daemon.c

```diff
@@ -72,6 +72,7 @@ char *opt_exports = "/etc/exports";
 int opt_cluster = FALSE;
 char *opt_cluster_path = "/";
 int opt_tcponly = FALSE;
+int opt_ipv4only = FALSE;
```

Comment:
Do commit message napsat, k čemu je to dobré - proč je potřeba tuhle volbu přidávat.